src/site/content/en/fast/preload-critical-assets/index.md

```diff
@@ -137,6 +138,36 @@ import(_/* webpackPreload: true */_ "CriticalChunk")
 
 If you are using an older version of webpack, use a third-party plugin such as [preload-webpack-plugin](https://github.com/GoogleChromeLabs/preload-webpack-plugin).
 
+## Effects of preloading on Core Web Vitals
+
+Preloading is a powerful performance optimization that has an effect on loading speed. Such optimizations can lead to changes in your site's [Core Web Vitals](/vitals/), and it's important to be aware them.
+
+### Largest Contentful Paint (LCP)
+
+Preloading has a powerful effect on [Largest Contentful Paint (LCP)](/lcp/) when it comes to fonts and images, as both images and text nodes can be [LCP candidates](/lcp/#what-elements-are-considered). Hero images and large runs of text that are rendered using web fonts can benefit significantly from a well-placed preload hint, and should be used when there are opportunities to deliver these important bits of content to the user faster.
+
+However, you want to be careful when it comes to preloading&mdash;and other optimizations! In particular, avoid preloading too many resources. When a resource is preloaded, its [resource priority](/prioritize-resources/) is effectively raised from whatever its initial priority would have been to "High" or "Highest". If too many resources are prioritized, effectively none of them are. The effects of excessive preload hints will be especially detrimental to those on slower networks where bandwidth contention will be more evident.
```

Comment:
This advice is good, but I don't think the reasoning is true anymore. Preloading used to bump the priority of the resource, but ever since Priority Hints were added, it no longer does. It just keeps the same priority it would have had without preload.

I think the primary reason to avoid over-preloading is because it leads to network contention.